src/deps/binary/base64-browser.js

```diff
@@ -3,6 +3,9 @@ var thisAtob = function (str) {
 };
 
 var thisBtoa = function (str) {
+  if (/[\u0080-\uFFFF]/.test(str)) {
+    str = unescape(encodeURIComponent(str));
+  }
```

Comment:
I'd also be interested to know: if the user provides a string which is already `unescape(encodeURIComponent())`d, would this break for them or would it still work?